tests/contrib/langchain/test_langchain_llmobs.py

```diff
@@ -122,6 +122,20 @@ def _invoke_chain(cls, chain, prompt, mock_tracer, cassette_name, batch=False):
         LLMObs.disable()
         return mock_tracer.pop_traces()[0]
 
+    def _embed_query(cls, embedding_model, query, mock_tracer, cassette_name):
+        LLMObs.enable(ml_app=cls.ml_app, integrations_enabled=False, _tracer=mock_tracer)
+        with get_request_vcr(subdirectory_name=cls.cassette_subdirectory_name).use_cassette(cassette_name):
+            embedding_model.embed_query(query)
```

Comment:
@Yun-Kim Not sure if there's a different method call for langchain v0. I couldn't find a difference in the public documentation though.

Reply:
I believe the methods are the same in both langchain v0 and v1, `embed_query()` and `embed_documents()`. We can keep this as is here.